remotes/docker/pusher.go

```diff
@@ -52,7 +52,11 @@ func (p dockerPusher) Push(ctx context.Context, desc ocispec.Descriptor) (conten
 	case images.MediaTypeDockerSchema2Manifest, images.MediaTypeDockerSchema2ManifestList,
 		ocispec.MediaTypeImageManifest, ocispec.MediaTypeImageIndex:
 		isManifest = true
-		existCheck = path.Join("manifests", desc.Digest.String())
+		if p.tag == "" {
```

Comment:
Do you have a coding style / policy on `len(p.tag) == 0` vs `== ""`?

Reply:
Policy is do the right way? ;)

Reply:
I generally prefer option A which makes sense in other languages too vs comparing to empty string. Just curious whether you had any opinions either way.

Reply:
Its less verbose and more straight forward this way.  I've never felt like we should write Go code like other languages, write the best code for the language you are working with.  I'm always taken aback when I see `len(string) == 0`, it takes extra processing time when I'm reading the code.